tests/test_dataclasses.py

```diff
@@ -199,21 +205,26 @@ class TestOffer:
     def setup_method(self):
         self.initial_data = {
             "id": uuid.uuid4(),
-            "creation_time": DateTime.now(),
-            "price": 10,
+            "creation_time": test_time_stamp(),
+            "price": 10.,
```

Comment:
Why did you add a floating point ?

Reply:
I can not remember, I think to conform to the constructor parameter typing.

Reply:
FYI: the `float` type also includes integers.

Reply:
I will remove it then.